10exercise.js

```diff
@@ -0,0 +1,8 @@
+const ages = [83, 53, 37, 29, 60, 30, 66, 19, 59, 41, 9, 64, 19, 80, 24, 53, 70, 1, 53, 40, 92, 4, 71, 65, 8, 2, 51, 80, 94, 37, 80, 64, 19, 6, 14];
+
+const sortedAges = ages.sort()
+const startParam = (sortedAges.length - 1 ) / 2;
```

Comment:
Good instinct to start by sorting! I encourage you to finish the solution! I would probably use Math.floor/ceil instead of splice since you just need to identify the element not remove it.